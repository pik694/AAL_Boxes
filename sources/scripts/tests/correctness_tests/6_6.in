6
24 97 78
90 35 1
69 58 26
20 98 99
26 63 92
62 41 91

