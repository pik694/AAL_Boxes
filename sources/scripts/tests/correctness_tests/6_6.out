1 4 2 5 0 3 
864844->757642
